config/retronas.json

```diff
@@ -1552,9 +1552,9 @@
                     "index":"04",
                     "title": "ROM import",
                     "description": "Import roms via Smokemonster SMDBs",
-                    "id":"romimport",
+                    "id":"romimport-menu",
                     "prompt": "Open",
-                    "type": "dialog_yn",
```

Comment:
we'll need to work on this menu, its pretty small on some term sizes so its not exactly apparent whats going on